tests/baselines/reference/autoAccessor3(target=es5).js

```diff
@@ -0,0 +1,61 @@
+//// [tests/cases/conformance/classes/propertyMemberDeclarations/autoAccessor3.ts] ////

```

Comment:
Just so I understand, the reason all of the baselines change is because we've enabled `noCheck` for _all_ of them?